README.md

```diff
@@ -52,7 +63,7 @@ import { NextcloudVuePlugin } from '@nextcloud/vue'
 Vue.use(NextcloudVuePlugin)
 ```
 
-## Development setup
+## 🧑‍💻 Development setup
```

Comment:
I would maybe wrap all this into a `Developing this library` or `Contributing`.
Meaning make this and all following h3.

I think it then makes clearer that this is not usage (app development) but for this library itself.